### DISCUSSION TITLE
Tests won't run on Windows

### DISCUSSION THREAD

**Discussion by ollyde** [avajs/ava#2757] [Q&A]
We have a set of standard tests. We even deleted them all down to what would be the AVA examples.

The tests are written in TypeScript with the extention ts, no tsx or jsx.

On Mac Big Sur it runs fine, no issues. (M1 Macbook Pro)
On Windows (multiple machines including one arm machine) it fails.

Running node v14.17.0.

Our config

```
module.exports = () => {
    return {
        extensions: [
            "ts"
        ],
        require: [
            "ts-node/register"
        ],
        files: [
            "src/tests/*"
        ],
        verbose: true,
        concurrency: 1,
        environmentVariables: {
            "GOOGLE_APPLICATION_CREDENTIALS": "service_account_dev.json",
            "TEST_MODE": "TRUE", 
            "NODE_ENV": "dev"
        },
    }
}
```

Windows  ❌
<img width="566" alt="Screenshot 2021-05-27 at 09 16 52" src="https://user-images.githubusercontent.com/6940726/119782547-9ce82f00-becc-11eb-89b3-17552dff0368.png">

Mac (The test is running) ✅
<img width="570" alt="Screenshot 2021-05-27 at 09 20 31" src="https://user-images.githubusercontent.com/6940726/119782706-cbfea080-becc-11eb-934b-a8667a19503b.png">

**Comment by novemberborn** [comment]
My best guess is that there's an error coming out of `ts-node` that somehow gets lost. Can you compile your project on Windows? Can you run it through `ts-node`? Can you get this to work if you run AVA against the compiled code, without `ts-node`?

**Reply by ollyde** [reply]
@novemberborn thanks for replying. Yes indeed we can compile it; but literally 5 minutes ago we discovered what the bug was.

We're not precompiling as we like to use the debugger via tests.

To fix it we had to increase the ``timeout`` value to 2 minutes then switch from Powershell to Bash on Windows.

It's still extremely slow on Windows only; there's only a couple 100 tests as well.
